deployment/frontend/src/components/search/TemporalCoverageFacet.tsx

```diff
@@ -79,7 +79,7 @@ export default function TemporalCoverageFacet({
                                         </div>
```

Comment:
Since you used "Since" and "After" on the last updated filter, maybe we should change the labels here as well?

Reply:
I don't think we need to make these the same. I see them as different concepts. "Temporal Coverage" is expected to be a range, so "Start" and "End" make sense here. For "Last Updated", "Since" and "Before" make more sense to me since it's not necessarily a range (though "Teporal Coverage" doesn't _have_ to be either, it is in the metadata). "Last Updated Since" and/or "Last Updated Before" is how I look at this one.